text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+* Personal website
+* GH handle (?)
+* Open for Work [boolean]
+* A photo / logo (depending on Individual / Organization type)
+* Org they are affiliated with (if Individual)
+* People they are affiliated with (if Org)
+* A list of open jobs (if they are Organization type)
+* A list of all the sites they’ve built.
+* A location with:
+    * A description (“London, UK”)
+    * Lat / lng
+
+_Data Format_: all creators should live in a creators.yml file, similar to sites.yml.
+
+**Jobs** have the following fields:
+* Title
+* Location
+* Organization (FK)
+* (Optional) salary low end
+* (Optional) salary high end
+* (Optional) salary currency
+* Contact email or application link. 
+* Excerpt (short version)
+* Description
+
+_Data Format_: Jobs should be stored in markdown files in a /jobs folder. Jobs should be stored in markdown files. Frontmatter should have all fields except description. Body of the file should be the description. 
+
+### Notes on long-term data strategy
+* Right now, the data is in YAML in the GitHub repo. At some point, we may move all of our data to a format with a UI making it easier for people to submit & edit their own data, without imposing a review burden on Gatsby maintainers
+* Option 1: use Airtable or a CMS
+    * Airtable doesn’t have an easy way to allow users to edit individual forms in a record. 
+    * We could build something on top with Lambdas, or utilize the Zapier => Airtable integration: https://support.airtable.com/hc/en-us/articles/115007655568-Using-forms-to-edit-tables-with-Zapier
+    * We could also make job postings go down after 60 days
+* Option 2: Build an application ourselves
+    * Could do this, but what’s the value here?
+    * We probably don’t want to spin up an application to do this
+    * Firebase as backend?
+* **Option 3: Stick with YAML in Github for now (what we’re doing)**
+    * We know this works, although it does add additional maintainer burden reviewing these files
+    * If we find a good global “application” type solution, we probably want to migrate the sites in the site showcase here as well. 
+    * **Almost certainly the right move for now.**
+* Option 4: Use seeker.company or another TBD service
+    * Upside: seeker.company provides a UI to allow people to create job postings, and a job board to list those postings. 
+    * Downside: No control over look and feel, not built with Gatsby, no way to customize submissions or link to organizations, no way to edit postings.
+# Drawbacks
+
+Why should we *not* do this? Please consider:
+
+- Time & effort
+- It will add more pages to .org to maintain
+- Maintainers will be getting PRs adding additional agencies / sites / companies
+- We could get spam submissions that degrade the site.
+
+# Alternatives
+
+We could use 3rd-party services to build these pages (eg jobs board) & maybe individual and community pages. They would not be sufficiently customizable and would look weird (are we a website building tool or not!?)
```

Comment:
Other alternatives:

- Find a third-party tool that manages this kind of info and exposes an API. This way we could render the site with Gatsby, but we're not open to data management headaches.
- Integrate this into the [Gatsby Dashboard](https://store.gatsbyjs.org/account/dashboard). Advantages include:
	- reuse existing auth flow/site foundation
	- pull a lot of this info from the dev's GitHub profile
	- potentially limit spam by requiring auth (could also add criteria, e.g. "must have contributed to Gatsby" — probably not a good idea, but possible)
	- create a canonical example of how to build a data management dashboard using Gatsby (practical implementation, potential for tutorials, etc.)

Reply:
Gatsby dashboard could be an interesting place to put this, because it already has auth. I think we're pretty sold on YAML first, but could be an interesting place to put it longer-term.